components/browser/engine-gecko-beta/src/test/java/mozilla/components/browser/engine/gecko/webnotifications/GeckoWebNotificationDelegateTest.kt

```diff
@@ -1,47 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-package mozilla.components.browser.engine.gecko.webnotifications
-
-import androidx.test.ext.junit.runners.AndroidJUnit4
-import mozilla.components.concept.engine.webnotifications.WebNotificationDelegate
-import mozilla.components.support.test.any
-import mozilla.components.support.test.mock
-import org.junit.Assert.assertEquals
-import org.junit.Test
-import org.junit.runner.RunWith
-import org.mockito.Mockito.doNothing
-import java.lang.IllegalStateException
-import org.mozilla.geckoview.WebNotification as GeckoViewWebNotification
-
-@RunWith(AndroidJUnit4::class)
```

Comment:
We don't need these tests anymore?

Reply:
We still have GeckoWebNotificationDelegate.kt

Reply:
@jonalmeida asked this in an earlier version of this PR, see here: https://github.com/mozilla-mobile/android-components/pull/8359